src/codewhisperer/commands/basicCommands.ts

```diff
@@ -189,3 +189,14 @@ export const notifyNewCustomizationsCmd = Commands.declare(
         notifyNewCustomizations().then()
     }
 )
+
+export const applySecurityFix = Commands.declare(
+    'aws.codewhisperer.applySecurityFix',
+    () => async (codeScanIssue: CodeScanIssue) => {
+        const activeEditor = vscode.window.activeTextEditor
+        if (activeEditor) {
+            const filePath = activeEditor.document.uri.fsPath
```

Comment:
What's the difference between this and `activeEditor.document.fileName`?

Reply:
Both are same. Shorthand notation for TextDocument.uri.fsPath. Updating to fileName